frontstage/views/secure_messaging.py

```diff
@@ -42,12 +42,24 @@
 def create_message(session):
     """Handles sending of new message"""
 
+    url = app.config['RM_CASE_GET_BY_PARTY'].format(app.config['RM_CASE_SERVICE'], session['user_uuid'])
+    # url = "http://localhost:8050/api/party-api/respondents/id/" + session['user_uuid']
+    collection_response = requests.get(url)
+    if collection_response.status_code != 200:
+        return redirect(url_for('error_bp.error_page'))
+    collection_response_json = collection_response.json()
+    collection_id = collection_response_json[0].get('id')
+    if collection_id:
+        collection_case = collection_id
+    else:
+        collection_case = None
```

Comment:
Could also have this as a separate function as we're using with reply as well but that could be done in that card.